templates/zerver/realm_deactivated.html

```diff
@@ -0,0 +1,15 @@
+{% extends "zerver/portico_signup.html" %}
+{# Displayed after a user attempts to sign up. #}
+
+{% block portico_content %}
+<div class="app portico-page">
+    <div class="app-main portico-page-container center-block flex full-page account-creation new-style">
+        <div class="inline-block">
+
+            <div class="white-box">
+                <p>{{ _("This realm has been deactivated.") }}</p>
```

Comment:
This should say "This Zulip organization has been deactivated.", since that's what we call it in user-facing strings.

Reply:
Yep.